src/coreclr/src/vm/method.cpp

```diff
@@ -4865,6 +4865,14 @@ bool MethodDesc::IsJitOptimizationDisabled()
 {
     WRAPPER_NO_CONTRACT;
 
+#ifdef FEATURE_INTERPRETER
+    static ConfigDWORD s_EnableJitOptimization;
+    if (s_EnableJitOptimization.val(CLRConfig::INTERNAL_EnableJitOptimization) != 0)
```

Comment:
How is this different from the existing `JitMinOpts` flag (below)? Can you just use the existing `JitMinOpts` to do what you need?